### PR TITLE
Adds monochromatic trait from /tg/

### PR DESCRIPTION
**Pull request by ghost**
just ports over the monochromatic trait by Xhuis since @JohnGinnane asked for it

### CODE REVIEW

File: hippiestation/code/datums/traits/neutral.dm [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+	name = "Monochromacy"
+	desc = "You suffer from full colorblindness, and perceive nearly the entire world in blacks and whites."
+	value = 0
+	medical_record_text = "Patient is afflicted with almost complete color blindness."
+
+/datum/trait/monochromatic/add()
+	trait_holder.add_client_colour(/datum/client_colour/monochrome)
+
+/datum/trait/monochromatic/post_add()
+	if(trait_holder.mind.assigned_role == "Detective")
```
Comment (JohnGinnane) [inline hippiestation/code/datums/traits/neutral.dm:12]: i love this touch

**Review by JohnGinnane — DISMISSED** [review]

File: hippiestation/code/datums/traits/neutral.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/datums/traits/neutral.dm:12]: i'm sure that was lifted from /vg/, they have a detective noire thing there too and it's slick

**Comment by JohnGinnane** [comment]
do we need head input for this

File: hippiestation.dme [hunk 2]
```diff
@@ -693,10 +693,10 @@
 #include "code\game\objects\effects\decals\decal.dm"
 #include "code\game\objects\effects\decals\misc.dm"
 #include "code\game\objects\effects\decals\remains.dm"
-#include "code\game\objects\effects\decals\Cleanable\aliens.dm"
-#include "code\game\objects\effects\decals\Cleanable\humans.dm"
```
Comment (Carbonhell) [inline hippiestation.dme:697]: aaaAAAAAA why does this shit keep appearing god damn it

**Comment by Carbonhell** [comment]
yes we do
+1

**Comment by ghost** [comment]
+1 great

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**